css/style.scss

```diff
@@ -0,0 +1,382 @@
+@import "fonts";
+@import "slider";
+
+
+$mainBgColor: rgb(45, 48, 58);
+$mainBgColorBorder: rgb(50, 55, 70);
+$redColor: rgb(255, 108, 100);
+
+$navTextColor: rgb(255, 255, 255);
+$darkTextColor: rgb(102, 109, 137);
+
+$darkTextColorHover: rgb(142, 151, 189);
+
+
+html {
+      scroll-behavior: smooth;
+}
+body {
+      display: flex;
+      justify-content: center;
+      flex-direction: column;
+      box-sizing: border-box;
+}
+
+//                                   СТИЛИ HEADER
+.shadow {
+      position: absolute;
+      top: 0;
+      left: 0;
+      width: 0%;
+      height: 0%;
+      z-index: 11;
+      background-color: rgba(45, 48, 58, 0);
+      transition: background-color 0.4s linear;
+}
+header {
+      position: fixed;
+      display: flex;
+      flex-direction: row;
+      justify-content: center;
+      border-bottom: 6px solid $mainBgColorBorder;
+      background-color: $mainBgColor;
+      height: 89px;
+      width: 100%;
+      z-index: 10;
+      top: 0;
+      left: 0;
+
+      section {

```

Comment:
Don't use elements as a target, replace to class name

Reply:
fixed